commands/list.js

```diff
@@ -9,7 +9,7 @@ var _ = require('lodash');
 var logger = require('../lib/logger');
 
 module.exports = new Command('list')
-  .description('list the Firebases to which you have access')
+  .description('list the Firebase projects you have access to')
```

Comment:
Grammatically "Firebase projects to which you have access" is correct...pedantry FTW!